const insertUser = async() => {
    try {
        
    } catch (error) {
        
    }
}